### PR TITLE
changed cluster topo to true

### PR DESCRIPTION
**Pull request by henderiw**
we have to set ClusterTopology=true to use the kind docker provider

### CODE REVIEW

File: cluster-capi/cluster-api-control-plane.yaml [hunk 1]
```diff
@@ -6663,7 +6663,7 @@ spec:
       - args:
         - --leader-elect
         - --metrics-bind-addr=localhost:8080
-        - --feature-gates=ClusterTopology=false,KubeadmBootstrapFormatIgnition=false,LazyRestmapper=false
+        - --feature-gates=ClusterTopology=true,KubeadmBootstrapFormatIgnition=false,LazyRestmapper=false
```
Comment (vjayaramrh) [inline cluster-capi/cluster-api-control-plane.yaml:6666]: I am guessing that the ClusterTopology=true activates/deploys the ClusterTopology Controller (https://cluster-api.sigs.k8s.io/developer/architecture/controllers/cluster-topology.html), right?

**Comment by henderiw** [comment]
yes and it is needed for docker kind with cluster api
https://cluster-api.sigs.k8s.io/user/quick-start.html#enabling-feature-gates

File: cluster-capi/cluster-api-control-plane.yaml (hunk 1 above)
Comment (johnbelamaric) [reply cluster-capi/cluster-api-control-plane.yaml:6666]: yes, it seems we need it - our v2 package has it and without it our kind provisioning fails

**Comment by johnbelamaric** [comment]
/lgtm

I need to merge this and #41 to get the e2e sandbox ready to merge.

**Comment by electrocucaracha** [comment]
Eventually this change needs to be included as part of the [upgrade script](https://github.com/nephio-project/nephio-example-packages/blob/main/update_packages.sh#L52)

**Pull request merged**